### PR TITLE
remove solrizer gem as it obscures more than it helps

### PR DESCRIPTION
**Pull request by ndushay**
The tests certainly failed until I got the code right, so I would say this has been well tested.

Making things simpler and more obvious feels like the right direction.

Plus, solrizer became part of ActiveFedora about 5 years ago and has had no maintenance.

### CODE REVIEW

File: lib/dor_indexing/indexers/collection_title_indexer.rb [hunk 1]
```diff
@@ -11,14 +11,16 @@ def initialize(cocina:, parent_collections:, **)
         @parent_collections = parent_collections
       end
 
-      # @return [Hash] the partial solr document for identifiable concerns
+      # @return [Hash] the partial solr document for collection title concerns
```
Comment (ndushay) [inline lib/dor_indexing/indexers/collection_title_indexer.rb:14]: copy pasta of former tautology?

File: lib/dor_indexing/indexers/collection_title_indexer.rb [hunk 2]
```diff
@@ -11,14 +11,16 @@ def initialize(cocina:, parent_collections:, **)
...
       def to_solr
         {}.tap do |solr_doc|
-          parent_collections.each do |related_obj|
-            coll_title = Cocina::Models::Builders::TitleBuilder.build(related_obj.description.title)
+          parent_collections.each do |collection_obj|
```
Comment (ndushay) [inline lib/dor_indexing/indexers/collection_title_indexer.rb:17]: here's an idea - let's use good variable names!

File: lib/dor_indexing/indexers/collection_title_indexer.rb [hunk 3]
```diff
@@ -11,14 +11,16 @@ def initialize(cocina:, parent_collections:, **)
...
+            coll_title = Cocina::Models::Builders::TitleBuilder.build(collection_obj.description.title)
 
-            # create/append collection_title_tesim and collection_title_ssim
-            ::Solrizer.insert_field(solr_doc, 'collection_title', coll_title, :stored_searchable, :symbol)
```
Comment (ndushay) [inline lib/dor_indexing/indexers/collection_title_indexer.rb:21]: :stored_searchable and :symbol told Solrizer to make two different flavors of fields.  Obvious, right?

**Review by jmartin-sul — APPROVED** [review]
sunset for solrizer 🌆 🌚

**Pull request merged**